tools/topology/sof/pipe-highpass-capture.m4

```diff
@@ -0,0 +1,123 @@
+# IIR highpass filter, volume control, and PCM
+#
+# Pipeline Endpoints for connection are :-
+#
+#  host PCM_C <-- B0 <-- Volume <-- B1 <-- EQ_IIR <-- B2 <-- source DAI0
+
+# Include topology builder
+include(`utils.m4')
+include(`buffer.m4')
+include(`pcm.m4')
+include(`pga.m4')
+include(`dai.m4')
+include(`mixercontrol.m4')
+include(`pipeline.m4')
+include(`bytecontrol.m4')
+include(`eq_iir.m4')
+
+#
+# Controls
+#
+# Volume Mixer control with max value of 32
+C_CONTROLMIXER(Master Capture Volume, PIPELINE_ID,
+	CONTROLMIXER_OPS(volsw,
+		256 binds the mixer control to volume get/put handlers,
+		256, 256),
+	CONTROLMIXER_MAX(, 80),
+	false,
+	CONTROLMIXER_TLV(TLV 80 steps from -50dB to +30dB for 1dB, vtlv_m50s1),
+	Channel register and shift for Front Left/Right,
+	LIST(`	', KCONTROL_CHANNEL(FL, 1, 0), KCONTROL_CHANNEL(FR, 1, 1)))
+
+#
+# Volume Configuration
+#
+
+define(MY_PGA_TOKENS, concat(`pga_tokens_', PIPELINE_ID))
+define(MY_PGA_CONF, concat(`pga_conf_', PIPELINE_ID))
+W_VENDORTUPLES(MY_PGA_TOKENS, sof_volume_tokens,
+LIST(`		', `SOF_TKN_VOLUME_RAMP_STEP_TYPE	"0"'
+     `		', `SOF_TKN_VOLUME_RAMP_STEP_MS		"400"'))
+
+W_DATA(MY_PGA_CONF, MY_PGA_TOKENS)
+
+#
+# IIR configuration
+#
+
+include(`eq_iir_coef_highpass_40hz_0db_48khz.m4')
+
+# EQ Bytes control with max value of 255
```

Comment:
Could you please double check - I learned recently, that m4 `define` statements generate empty lines in .conf, so, using `#` comments with them produces a stand-alone comment, attached to nothing. So it's better to use `dnl` to comment `define`s

Reply:
The comment need to be attached to following SectionControlBytes."<unique instance ID>" {} so in my opinion OK.